File: CursoPOOUber/Csharp/Driver.cs
using AccountNS;

namespace DriverDetails
{
    //Definition of the child "Driver" class, which inherits from "Car" class.
    public class Driver : Account
    {
        public Driver(string name, string document) : base (name, document)
        {

        }

        //This method is overwriting the method assigned in its parent class.
        public override void PrintDataDriver() {
            base.PrintDataDriver();
        }        
    }
}

File: CursoPOOUber/PHP/Driver.php
<?php
require_once('account.php');

class Driver extends Account {

    public function __construct($name, $document) {
        $this->name = $name;
        $this->document = $document;
    }

    public function printDataDriver() {
        echo "Name: " + $this->name + " Document: " + $this->document + " E-mail: " + $this->email + " Password: " + $this->password;
    }
}
?>